SPECS/kernel/kernel.spec

```diff
@@ -251,6 +251,18 @@ Patch1209:      CVE-2021-45402.nopatch
 Patch1210:      CVE-2021-3752.nopatch
 Patch1211:      CVE-2021-20322.nopatch
 Patch1212:      CVE-2021-43976.nopatch
+# CVE-2021-4090 - Introducing commit not in stable tree. No fix necessary at this time.
```

Comment:
Why are we commenting on the reason for the nopatch in both the specfile and the .nopatch file?